src/StackExchange.Redis/ConnectionMultiplexer.cs

```diff
@@ -480,17 +481,17 @@ void Broadcast(ReadOnlySpan<ServerEndPoint> serverNodes)
                 }
             }
 
-            // Send a message before it happens - because afterwards a new slave may be unresponsive
+            // Send a message before it happens - because afterwards a new replica may be unresponsive
             Broadcast(nodes);
 
-            if ((options & ReplicationChangeOptions.EnslaveSubordinates) != 0)
+            if ((options & ReplicationChangeOptions.ReplicateToSubordinates) != 0)
```

Comment:
This one seems odd (but works) to me, the action isn't *from the master* as the verbage hints it, it's an action *on the subordinates*. Trying to think of a better name while reading this...

Reply:
Maybe `ReplicateToOtherEndpoints`? That's really what's happening here.

Reply:
done; also added format test